sale_medical_prescription/models/medical_physician.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class MedicalPhysician(models.Model):
+    _inherit = 'medical.physician'
+
+    is_verified = fields.Boolean(
```

Comment:
Add to readme:
`* Move duplicated `is_verified`, `verified_by_id`, `verified_date` & related methods to `medical.entity.abstract`